src/Components/Facility/models.tsx

```diff
@@ -229,3 +226,430 @@ export type ICD11DiagnosisModel = {
+  district_object: DistrictModel;
+  state_object: StateModel;
+  pf_endpoint: string;
+  pf_p256dh: string;
+  pf_auth: string;
+};
+
+export type DeleteModel = {
+  detail?: string;
+};
+
+export type IUserFacilityRequest = {
+  username?: string;
+  first_name?: string;
+  last_name?: string;
+  email?: string;
+  doctor_qualification?: string;
+  doctor_experience_commenced_on?: string;
+  doctor_medical_council_registration?: string;
+  home_facility?: string;
+  weekly_working_hours?: number;
+  phone_number?: string;
+  alt_phone_number?: string;
+  gender?: number;
+  age?: number;
+};
+
+export type IUserFacilityResponse = IUserListFacilityResponse;
+
+export type IFacilityUserResponse = {
+  count: number;
+  next: string | null;
+  previous: string | null;
+  results: {
+    id: number;
+    first_name: string;
+    username: string;
+    email: string;
+    last_name: string;
+    alt_phone_number: string;
+    user_type: number;
+    last_login: string;
+    home_facility_object: HomeFacilityModel;
+    doctor_qualification: string;
+    doctor_experience_commenced_on: string;
+    doctor_medical_council_registration: string;
+    skills: {
+      id: string;
+      name: string;
+      description: string;
+    }[];
+  }[];
+};
+
+export type ILocalBodyResponse = StateModel & {
+  body_type: number;
+  localbody_code: string;
+  district: number;
+};
+
+export interface InventoryItemObjectModel {
+  id: number;
+  default_unit: UnitModel;
+  allowed_units: UnitModel[];
+  tags: UnitModel[];
+  name: string;
+  description: string;
+  min_quantity: number;
+}
+
+export type IInventorySummaryResponse = PaginatedResponse<{
+  id: string;
+  item_object: InventoryItemObjectModel;
+  unit_object: UnitModel;
+  created_date: string;
+  quantity: number;
+  is_low: boolean;
+  item: number;
+}>;
+
+export type IInventoryLogResponse = Omit<
+  IInventorySummaryResponse,
+  "results"
+> & {
+  results: {
+    id: string;
+    item_object: InventoryItemObjectModel;
+    unit_object: {
+      id: number;
+      name: string;
+    };
+    external_id: string;
+    current_stock: number;
+    quantity_in_default_unit: number;
+    is_incoming: boolean;
+    probable_accident: boolean;
+    created_by: number;
+  }[];
+};
+
+export type IFaciclityMinimumQuantityRequest = {
+  min_quantity?: number;
+  item?: number;
+};
+export type IFaciclityMinimumQuantityResponse = {
+  id: string;
+  item_object: InventoryItemObjectModel;
+  created_date: string;
+  min_quantity: number;
+  item: number;
+};
+
+interface FacilityObjectModel {
+  id: string;
+  name: string;
+  local_body: number;
+  district: number;
+  state: number;
+  ward_object: WardModel;
+  local_body_object: { id: number } & LocalBodyModel;
+  district_object: DistrictModel;
+  state_object: StateModel;
+  facility_type: string;
+  read_cover_image_url: string;
+  features: number[];
+  patient_count: string;
+  bed_count: string;
+}
```

Comment:
Why redefine `FacilityModel` in so many different ways? Just use the one already defined `FacilityModel` right?